src/pages/components/Navbar/SubMenu.js

```diff
@@ -30,12 +30,39 @@ const SubMenu = ({ isVisible, transparent }) => {
   useEffect(() => {
     changeLanguage(language);
   }, [language]);
+  // Check if the viewport matches a mobile screen width
+    const mediaQuery = window.matchMedia('(max-width: 767px)');
+    const handleMediaQueryChange = (event) => {
+      setIsMobile(event.matches);
+    };
+
+    const [isMobile, setIsMobile] = useState(false);
+
+  useEffect(() => {
+    const mediaQuery = window.matchMedia('(max-width: 767px)');
+    const handleMediaQueryChange = (event) => {
+      setIsMobile(event.matches);
+    };
+
+    // Initial check
+    handleMediaQueryChange(mediaQuery);
+
+    // Listen for changes in viewport width
+    mediaQuery.addEventListener('change', handleMediaQueryChange);
+
+    // Clean up the event listener
+    return () => {
+      mediaQuery.removeEventListener('change', handleMediaQueryChange);
+    };
+  }, []); // Empty dependency array to run the effect only once
```

Comment:
none of this is necessary since you can just use [responsive TailwindCSS classnames](https://tailwindcss.com/docs/responsive-design)
```suggestion
```

Reply:
@PeaceOloruntoba please make this change - we don't need this extra code